content/applications/finance/sign.rst

```diff
@@ -101,71 +139,124 @@ template.
-itsme®`.
+Itsme® authentication can be used to allow signatories to provide their identity using itsme®. This
+feature is only available in **Belgium** and the **Netherlands**.
 
-Then, go to :menuselection:`Sign --> Configuration --> Roles`. Click in the :guilabel:`Extra
-Authentication Step` column for the role, and select :guilabel:`Via itsme®`.
+The feature can be enabled in :guilabel:`Sign Settings` and applies automatically to the
+:guilabel:`Customer (identified with itsme®)` role. To enable it for other roles, go to
+:menuselection:`Sign --> Configuration --> Roles`. Click in the :guilabel:`Extra Authentication
+Step` column for the role, and select :guilabel:`Via itsme®`.
 
-Go to the document that needs to be signed and add the :guilabel:`Signature` field. Switch the role
-to :guilabel:`customer (identified with itsme®)`, and click :guilabel:`Validate`, and
-:guilabel:`Send`.
+Go to the document that needs to be signed and add the :guilabel:`Signature` field. Switch to any
+role configured to use the feature, and click :guilabel:`Validate` and :guilabel:`Send`.
 
 .. image:: sign/itsme-identification.png
-   :align: center
    :alt: select customer identified with itsme®
 
-The person signing the document fills in the :guilabel:`Signature` field and clicks
-:guilabel:`Validate & Send Completed Document`. A :guilabel:`Final Validation` page pops up, and the
-person must authenticate with itsme®.
+Upon signing the document, the signer completes the :guilabel:`Signature` field and proceeds by
+clicking on :guilabel:`Validate & Send Completed Document`, triggering a
+:guilabel:`Final verification` page where authentication via itsme® is required.
 
-.. note::
-   This feature is only available in Belgium and the Netherlands.
+Signatory hash
+==============
+
+Each time someone signs a document, a hash - a unique ID number - is generated to ensure
+traceability.
+
+A visual security frame displaying the beginning of the hash is added to the signatures. Internal
+users can hide or show it by turning the :guilabel:`Frame` option on or off when signing the
+document.
+
+.. image:: sign/sign-hash.png
+   :alt: Adding the visual security frame to a signature.
 
 .. _sign/field-types:
 
-Field Types
+Tags
+====
+
+Tags can be used to categorize and organize documents, allowing users to search for and filter
+documents based on specific criteria quickly.
+
+You can manage tags by going to :menuselection:`Configuration --> Tags`. To create a tag, click
+:guilabel:`New`. On the new line, add the :guilabel:`Tag Name` and select a :guilabel:`Color Index`
+for your tag.
+
+To apply a tag to a document, use the dropdown list available in your document.
+
+Sign order
+==========
+
+When a document needs to be signed by different parties, the signing order lets you control the
+order in which your recipients receive it for signature.
+
+By going to :menuselection:`Configuration --> Settings`, you can :guilabel:`Enable Signing Order`.
```

Comment:
no longer needed in 17 btw, signing order is always available as soon as you have more than 1 signer
![image](https://github.com/odoo/documentation/assets/9530767/b678f6f0-a53b-43d5-ac8e-0517191f8bf3)

Reply:
![image](https://github.com/odoo/documentation/assets/9530767/4f49aa81-e521-426c-891c-5888db7c957d)

Reply:
I keep it in mind for V17, thanks!